### ISSUE TITLE
No suggestions for the if condition expressions in VSCode

### ISSUE THREAD

**Issue by anupama-pathirage** [ballerina-platform/ballerina-lang#17363]
**Description:**

$subject with the below code

```ballerina
public function main(string... args) {
    boolean b = false;
    if ()
}
```

```
[Error - 4:46:13 PM] Operation 'text/completion' failed! {uri: /Volumes/work/Ballerina/TestBallerina/TestAlpha/BBG/testProject/restful-service/guide/restful-service/src/restful_service/test.bal, line: 3, col:9, error: java.lang.NullPointerException, org.wso2.ballerinalang.compiler.semantics.analyzer.SymbolResolver.getAllVisibleInScopeSymbols(SymbolResolver.java:1182)}
java.lang.NullPointerException
	at org.wso2.ballerinalang.compiler.semantics.analyzer.SymbolResolver.getAllVisibleInScopeSymbols(SymbolResolver.java:1182)
	at org.ballerinalang.langserver.completions.TreeVisitor.resolveAllVisibleSymbols(TreeVisitor.java:829)
	at org.ballerinalang.langserver.completions.TreeVisitor.visit(TreeVisitor.java:197)
	at org.wso2.ballerinalang.compiler.tree.BLangPackage.accept(BLangPackage.java:160)
	at org.ballerinalang.langserver.completions.util.CompletionUtil.resolveSymbols(CompletionUtil.java:58)
	at org.ballerinalang.langserver.BallerinaTextDocumentService.lambda$completion$0(BallerinaTextDocumentService.java:180)
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1590)
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1582)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```
<img width="757" alt="image" src="https://user-images.githubusercontent.com/16300038/62366651-22a32980-b545-11e9-89cb-df9c70d8e439.png">

**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by MaryamZi** [comment]
Same with the IntelliJ IDEA plugin too.

**Comment by nadeeshaan** [comment]
This has been fixed in v1.0.0 onwards, hence closing the issue
